chore: saml acs url is now on the project slug (#2407)
ory/docs: docs/kratos/organizations/organizations.mdx

</Tabs>
```

> The SAML application callback URL to set at our SAML Identity Provider is:

Reviewer: ```suggestion
The SAML application callback URL to set as the SAML Identity Provider is:
```

> `https://your-slug.projects.oryapis.com/saml/api/oauth/saml`

Reviewer: ```suggestion
`https://$PROJECT_SLUG.projects.oryapis.com/saml/api/oauth/saml`
```
